<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BOMExportDialog</class>
 <widget class="QDialog" name="BOMExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export CSV</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item>
    <widget class="QWidget" name="m_info_widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Informations disponibles</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Informations à exporter</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QListWidget" name="m_var_list"/>
      </item>
      <item row="1" column="2">
       <widget class="QListWidget" name="m_choosen_list"/>
      </item>
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_up_pb">
          <property name="toolTip">
           <string>Monter la sélection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qelectrotech.qrc">
            <normaloff>:/ico/16x16/go-up.png</normaloff>:/ico/16x16/go-up.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_add_pb">
          <property name="toolTip">
           <string>Ajouter la sélection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qelectrotech.qrc">
            <normaloff>:/ico/16x16/list-add.png</normaloff>:/ico/16x16/list-add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_remove_pb">
          <property name="toolTip">
           <string>Supprimer la sélection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qelectrotech.qrc">
            <normaloff>:/ico/16x16/list-remove.png</normaloff>:/ico/16x16/list-remove.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_down_pb">
          <property name="toolTip">
           <string>Descendre la sélection</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../qelectrotech.qrc">
            <normaloff>:/ico/16x16/go-down.png</normaloff>:/ico/16x16/go-down.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_parametre_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Type d'éléments</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="4" column="0">
          <widget class="QCheckBox" name="m_simple_cb">
           <property name="text">
            <string>Simples</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="m_all_cb">
           <property name="text">
            <string>Tous</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="m_coil_cb">
           <property name="text">
            <string>Contacteurs et relais</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="m_button_cb">
           <property name="text">
            <string>Boutons et commutateurs</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="m_terminal_cb">
           <property name="text">
            <string>Borniers</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="m_protection_cb">
           <property name="text">
            <string>Organes de protection</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_format_as_gb">
     <property name="title">
      <string>Mise en page</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QCheckBox" name="m_include_header_cb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Inclure les en-têtes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="m_format_as_nomenclature_rb">
        <property name="toolTip">
         <string>Chaque élément portant la même référence sera listé</string>
        </property>
        <property name="text">
         <string>Formater en tant que nomenclature</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="m_format_as_bom_rb">
        <property name="toolTip">
         <string>Une même référence utilisé par plusieurs éléments ne sera listé qu'une fois</string>
        </property>
        <property name="text">
         <string>Formater en tant que liste de matériel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="m_load_pb">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Ouvrir la configuration sélectionné</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../qelectrotech.qrc">
          <normaloff>:/ico/16x16/folder-open.png</normaloff>:/ico/16x16/folder-open.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="m_conf_cb">
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="m_save_name_le"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="m_save_current_conf_pb">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Sauvegarder la configuration actuelle</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../qelectrotech.qrc">
          <normaloff>:/ico/16x16/document-save.png</normaloff>:/ico/16x16/document-save.png</iconset>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="m_edit_sql_query_cb">
       <property name="toolTip">
        <string>Requête SQL personnalisée</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Requête SQL :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_sql_query">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qelectrotech.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_button_box</sender>
   <signal>accepted()</signal>
   <receiver>BOMExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_button_box</sender>
   <signal>rejected()</signal>
   <receiver>BOMExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_var_list</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>m_add_pb</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_choosen_list</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>m_remove_pb</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
